gatsby-node.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+const fetch = require('node-fetch')
+
+module.exports.createPages = async ({ reporter, actions }) => {
+  if (process.env.SECRET_PAGE_CONTENTS_URL) {
+    reporter.info('Building secret page')
+
+    let pageContents
+
+    try {
+      pageContents = await fetch(
+        process.env.SECRET_PAGE_CONTENTS_URL,
```

Comment:
@exvuma any URL that responds with markdown works here. So we could just use a github gist to update content.